internal/pkg/cli/deploy.go

```diff
@@ -47,11 +48,14 @@ func newDeployOpts(vars deployWkldVars) (*deployOpts, error) {
 	if err != nil {
 		return nil, fmt.Errorf("new config store: %w", err)
 	}
-
 	ws, err := workspace.New()
 	if err != nil {
 		return nil, fmt.Errorf("new workspace: %w", err)
 	}
+	d, err := describe.NewAppDescriber(vars.appName)
+	if err != nil {
+		return nil, fmt.Errorf("new app describer for application %s: %v", vars.name, err)
```

Comment:
nit: just wondering if this should be:
```suggestion
		return nil, fmt.Errorf("new app describer for application %s: %w", vars.name, err)
```